Mathlib/LinearAlgebra/TensorProduct/Basic.lean

```diff
@@ -1111,20 +1141,31 @@ theorem tensorTensorTensorAssoc_symm_tmul (m : M) (n : N) (p : P) (q : Q) :
 end TensorProduct
 
 open scoped TensorProduct
+
 namespace LinearMap
 
 variable {N}
 
-/-- `lTensor M f : M ⊗ N →ₗ M ⊗ P` is the natural linear map induced by `f : N →ₗ P`. -/
+/-- `LinearMap.lTensor M f : M ⊗ N →ₗ M ⊗ P` is the natural linear map
+induced by `f : N →ₗ P`. -/
 def lTensor (f : N →ₗ[R] P) : M ⊗[R] N →ₗ[R] M ⊗[R] P :=
   TensorProduct.map id f
 #align linear_map.ltensor LinearMap.lTensor
 
-/-- `rTensor f M : N₁ ⊗ M →ₗ N₂ ⊗ M` is the natural linear map induced by `f : N₁ →ₗ N₂`. -/
+/-- `LinearMap.rTensor M f : N₁ ⊗ M →ₗ N₂ ⊗ M` is the natural linear map
+induced by `f : N₁ →ₗ N₂`. -/
 def rTensor (f : N →ₗ[R] P) : N ⊗[R] M →ₗ[R] P ⊗[R] M :=
   TensorProduct.map f id
 #align linear_map.rtensor LinearMap.rTensor
 
+theorem _root_.TensorProduct.coe_congr_left_refl (f : N ≃ₗ[R] P) :
+    TensorProduct.congr (.refl R M) f = lTensor M (f : N →ₗ[R] P) :=
```

Comment:
I think this should become two lemmas: `TensorProduct.congr (.refl R M) f = f.lTensor M` and `f.lTensor M = lTensor M (f : N →ₗ[R] P)` (if it fails to elaborate you may add type ascription to coerce LHS to LinearMap).

Reply:
I think the first is just the definition of `LinearEquiv.lTensor`, so `rw [← LinearEquiv.lTensor]` should work. The second is `LinearEquiv.coe_lTensor` I added.

Reply:
Perhaps `TensorProduct.coe_congr_left_refl` is not useful at this point, so maybe we can remove it? Since when we have `LinearEquiv.lTensor`, we don't write `TensorProduct.congr (.refl R M) f` anymore, but write `f.lTensor M` instead.

Reply:
> I think the first is just the definition of LinearEquiv.lTensor, so rw [← LinearEquiv.lTensor] should work. The second is LinearEquiv.coe_lTensor I added.

I see!

> Perhaps TensorProduct.coe_congr_left_refl is not useful at this point, so maybe we can remove it? Since when we have LinearEquiv.lTensor, we don't write TensorProduct.congr (.refl R M) f anymore, but write f.lTensor M instead.

Yeah I agree we should remove it.